components/SearchResultsList.vue

```diff
@@ -65,6 +65,7 @@
                                 : searchResult.facilities[0]?.nameEn"
                             :specialties="searchResult.professional.specialties"
                             :spoken-languages="searchResult.professional.spokenLanguages"
+                            :updated-date="searchResult.professional.updatedDate"
```

Comment:
product question: Instead of on every result in the list, should we show this in the details instead? 

I can think of two reasons to move it into the details section:
1) the cards will get bloated if we keep adding details
2) The card should have the most relevant result info and I don't think people should pick the doctor based on updated date. It's relevant info, but it doesn't always reflect quality or accuracy of the search result

cc @NabbeunNabi @theyokohamalife